lib/i18n-ai/clients/open_ai_client.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require "openai"
+require_relative "base_client"
+
+module I18nAi
+  module Clients
+    class OpenAiClient < BaseClient
+      def initialize
+        super()
```

Comment:
same comment if we can remove `()`. will put rubocop github action soon so it should handle these lint style feedback.